VTOLVR-ModLoader/MainWindow.xaml.cs

```diff
@@ -108,12 +107,14 @@ private async void WaitAsync()
         {
             await Task.Delay(500);
 
-            if (args.Length == 2 && root.Contains("System32"))
+            if (args.Length == 2 && root.ToLower().Contains("system32"))
                 URICheck();
             else
+            {
                 CheckBaseFolder();
+                GetData();
```

Comment:
We still need to call get data if they have opened the launcher through the one-click install because the update feed won't update and the launcher won't check if there any updates as well.

Reply:
Just move the GetData back out the else and it will be good 😄

Reply:
But GetData() is also called at the end of URICheck().

That was causing a crash, should I remove it from the else and from URICheck() or leave it as it is?

Reply:
ah. Yeah remove it from the URICheck() and the else then we should be good.